chapters/extensions/VK_KHR_multiview.md

```diff
@@ -0,0 +1,13 @@
+# VK_KHR_multiview
+
+> Promoted to core in Vulkan 1.1
+>
+> [SPV_KHR_multiview](https://htmlpreview.github.io/?https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/KHR/SPV_KHR_multiview.html)
+>
+> [GLSL - GL_EXT_multiview](https://github.com/KhronosGroup/GLSL/blob/master/extensions/ext/GL_EXT_multiview.txt)
+
+Multiview is a rendering technique originally designed for VR where it is more efficient to record a single set of commands to be executed with slightly different behavior for each "view". This is all done during Renderpass creation with the `VkRenderPassMultiviewCreateInfo` passed into `VkRenderPassCreateInfo::pNext`.
```

Comment:
"This" as in "recording a single set of commands" is not all done during render pass creation. Maybe "The multiview configuration is specified during .." ?

Reply:
is it possible to make the PR with any changes you want as the PR review period ended and happy to just accept any minor tweaks you have like these